le_utils/constants/file_types.py

```diff
@@ -10,6 +10,7 @@
 HTML5 = "html5"
 THUMBNAIL = "thumbnail"
 SUBTITLES = "subtitles"
+SLIDESHOW = "slideshow_image"
```

Comment:
Would it make sense to either call the variable `SLIDESHOW_IMAGE` or the value to be `slideshow` to keep the mapping more consistent?

Reply:
+1 for change constant name from `SLIDESHOW` to `SLIDESHOW_IMAGE`